xlators/protocol/server/src/server.c

```diff
@@ -1813,12 +1830,21 @@ struct volume_options server_options[] = {
     {.key = {"inode-lru-limit"},
      .type = GF_OPTION_TYPE_INT,
      .min = 0,
-     .max = 1048576,
+     .max = 67108864,
      .default_value = "16384",
      .description = "Specifies the limit on the number of inodes "
                     "in the lru list of the inode cache.",
      .op_version = {1},
      .flags = OPT_FLAG_SETTABLE | OPT_FLAG_DOC},
+    {.key = {"inode-table-size"},
```

Comment:
Why do we need to declare a new option inode-table-size, i think inode-table-size is already present, We can pass as an argument inode_table_size to the process.

Reply:
How would you pass that to a brick?

Reply:
So either modify glusterd or add an option?  Isn't then just adding an option simpler because that's how most of the rest of the system is managed anyway?

Please note, just trying to understand the logic here.

If adding options to glusterd is the way to go - is there an example mechanism I can follow?

Reply:
> So either modify glusterd or add an option? Isn't then just adding an option simpler because that's how most of the rest of the system is managed anyway?
> 
> Please note, just trying to understand the logic here.
> 
> If adding options to glusterd is the way to go - is there an example mechanism I can follow?

I still believe the best way is to use an existing option(inode-table-size) instead of creating a new argument.To 
configure the same via glusterd you  can declare an option at glusterd.c option table something like 
 working-directory and save the configured value in glusterd_conf and access the same value via glusterd_conf_t during brick start in glusterd_volume_start_glusterfs and pass as an argument. 
  
  @xhernandez @aravindavk  Can you please share your view about the same.